index.html

```diff
@@ -67,7 +67,7 @@ <h1>B&auml;rner Go Meetup</h1>
       </li>
       <li>
         To contact us, give feedback or just hang out, check our
-        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel</a>.
+        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel <code>#b&auml;rner-go</code></a> on Gopher Slack.<br/>Not yet hanging out there, get an <a href="https://invite.slack.golangbridge.org/">invite</a> and join.
```

Comment:
Does the line break work with the layout or is it a bit strange in a list?

Possible alternative:
```suggestion
        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel <code>#b&auml;rner-go</code></a> on Gopher Slack (not yet hanging out there? get an <a href="https://invite.slack.golangbridge.org/">invite</a> and join).
```

Reply:
Funny, I had it like this first and then though, that parenthesis make the text unnecessarily hard to read (looks more like code than prose). What do the others think?

Reply:
Maybe the good compromise is to keep it as a regular sentence (without the parenthesis) but add the questionmark.

```suggestion
        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel <code>#b&auml;rner-go</code></a> on Gopher Slack.<br/>Not yet hanging out there? Get an <a href="https://invite.slack.golangbridge.org/">invite</a> and join.
```